tests/fragments/test_pagination_page_size.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2021  Google LLC
```

Comment:
This fragment proto is redundant because we're already running showcase tests which uses this proto with `page_size`: https://github.com/googleapis/gapic-showcase/blob/5a47589b6ef373fcd55c2d5545b0a902ea570dd9/schema/google/showcase/v1beta1/echo.proto#L289C9-L289C27

Reply:
Good call. Will remove.

Reply:
@parthea 

Resolved, removed file.